tower-hedge/src/rotating_histogram.rs

```diff
@@ -1,8 +1,8 @@
 extern crate tokio_timer;
 
 use hdrhistogram::Histogram;
+use log::trace;
```

Comment:
tracing?

Reply:
I want to change to `tracing` in a separate PR.